src/utils/schema/resolvers/Query/getFilter.ts

```diff
@@ -345,7 +345,9 @@ const buildMongoFilter = (
             }
           }
           case 'lte': {
-            if (isNaN(intValue)) {
+            if (DATE_TYPES.includes(type)) {
+              return { [fieldName]: { $lte: endDate } };
+            } else if (isNaN(intValue)) {
```

Comment:
Looks right,
I think we also want to apply the corresponding logic to `case 'lte'` a few lines below!

(also applies for the other getFilter.ts file)

Reply:
@Matthis-M-ReliefApps  do you mean `case 'gte'` right?

Reply:
ofc, my bad!